bokehjs/src/coffee/api/plugin.coffee

```diff
@@ -0,0 +1,27 @@
+
+define [
+  "underscore"
+  "jquery"
+  "common/logging"
+  "./figure"
+  "./helpers"
+], (_, $, Logging, figure, helpers) ->
+
+  logger = Logging.logger
+  show = helpers.show
+
+  _api = {
+    "figure": figure
+  }
+
+  $.fn.bokeh = (type, args) ->
```

Comment:
On a side note. I universally hate creating APIs this way. If, for some reason, we really want to have a jquery plugin, then we should first create a in-bokeh well defined API, like `Bokeh.figure({...})` (or `Bokeh.API.figure({...})`) and then we can expose a jquery plugin, by making user include a special/additional javascript file, say `jquery.bokeh.js`.